pkg/identity/identity.go

```diff
@@ -30,6 +32,13 @@ type Identity struct {
 	LabelsSHA256 string `json:"labelsSHA256"`
 }
 
+// IPIdentityPair is a pairing of an IP and the security identity to which that
+// IP corresponds.
```

Comment:
Add a comment to indicate that this struct is stored in the kvstore and subject to backwards compatibility guarantees.

Similar to the warning in `type Endpoint struct`

Reply:
Done.